src/backend/core/models.py

```diff
@@ -441,6 +441,62 @@ def email_user(self, subject, message, from_email=None, **kwargs):
             raise ValueError("You must first set an email for the user.")
         mail.send_mail(subject, message, from_email, [self.email], **kwargs)
 
+    def get_abilities(self):
+        """
+        Return the user permissions at the application level (ie feature availability).
+
+        Note: this is for display purposes only for now.
```

Comment:
one of the main interests of this pattern was unicity of the information used for display and for check in permissions. Do you think it makes sense here?